dominion/dominion_ui/src/components/Modal/PortalChooser.js

```diff
@@ -120,90 +118,81 @@ const styles = theme => ({
   }
 });
 
-function PortalChooser({ classes, isOpen, handleClose }) {
+function PortalChooser({ classes, children, handleClose }) {
   const countries = window.dominion_countries;
   return (
-    <Modal isOpen={isOpen}>
+    <Grid
+      container
+      justify="flex-start"
+      direction="row"
+      className={classes.grid}
+    >
+      {children}
       <Grid
-        container
-        justify="flex-start"
-        direction="row"
-        className={classes.grid}
+        xs={12}
+        sm={12}
+        md={5}
+        lg={5}
+        xl={5}
```

Comment:
The above can be simplified to `<Grid xs={12} md={5}>`. You only have to specify the size **when** different from the preceding lower size(s) I think... _mobile first design/less is more_.